stetho/src/main/java/com/facebook/stetho/inspector/network/DownloadingAsyncPrettyPrinterFactory.java

```diff
@@ -21,8 +21,7 @@
  */
 public abstract class DownloadingAsyncPrettyPrinterFactory implements AsyncPrettyPrinterFactory {
 
-  private final static ExecutorService sExecutorService =
-      AsyncPrettyPrinterExecutorHolder.sExecutorService;
+  private static ExecutorService sExecutorService;
```

Comment:
delete this, and always call `AsyncPrettyPrinterExecutorHolder.getExecutorService()`